test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -280,3 +280,17 @@ Describe "Remoting loopback tests" -Tags @('CI', 'RequireAdminOnWindows') {
         $result | Should Be 100
     }
 }
+
+Describe "Verify remoting cmdlets that should not be implemented under non-windows platform" -Tags @("CI") {
+    It "Get-PSHostProcessInfo should be removed under non-windows platform" -Skip $IsWindows {
```

Comment:
We already have the test in [DefaultCommands.Tests.ps1](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/Basic/DefaultCommands.Tests.ps1#L277)

Reply:
As @iSazonov pointed out, the test `DefaultCommands.Tests.ps1` covers available cmdlets on different plats by default, can you please update that test?